pkg/recipe/worker.go

```diff
@@ -29,13 +29,21 @@ func (wq *WorkQueue) AddWork(work interface{}) {
 }
 
 func (wq *WorkQueue) Work() {
-	frames := frame.Factory(frame.Config{
-		Lines:         0,
-		HasHeader:     false,
-		HasFooter:     false,
-		TrailOnRemove: true,
+	fr := frame.New(frame.Config{
+		Lines:          0,
+		HasHeader:      true,
+		HasFooter:      true,
+		TrailOnRemove:  true,
+		PositionPolicy: frame.FloatFree,
+		ManualDraw:     false,
 	})
-	fr := frames[0]
+
+	fr.Header().WriteString("This is the best header ever!")
```

Comment:
is this just for testing purposes right now?  I know this used to be in the examples

Reply:
yeah, this should have been scrubbed...